tests/kafkatest/services/zookeeper.py

```diff
@@ -220,16 +220,15 @@ def create(self, path, chroot=None, value=""):
         output = self.nodes[0].account.ssh_output(cmd)
         self.logger.debug(output)
 
-    def describe(self, topic):
+    def describeUsers(self):
```

Comment:
replace the original `describe` topic method with `describeUsers` since describe topic via zk is not supported anymore.